x/wasm/types/params.go

```diff
@@ -195,6 +201,39 @@ func (a AccessConfig) ValidateBasic() error {
 	return sdkerrors.Wrapf(ErrInvalid, "unknown type: %q", a.Permission)
 }
 
+func validateGasMultiplier(i interface{}) error {
+	a, ok := i.(uint64)
+	if !ok {
+		return sdkerrors.Wrapf(ErrInvalid, "type: %T", i)
+	}
+	if a == 0 {
+		return sdkerrors.Wrap(ErrInvalid, "must be greater 0")
```

Comment:
Minor typo, insert "than" or:
```suggestion
		return sdkerrors.Wrap(ErrInvalid, "must be positive")
```
It may apply to other lines.